.env.example

```
@@ -9,4 +8,5 @@ FEED_BACK_EMAIL=___
 FEED_BACK_PASSWORD=___
 FEED_BACK_SERVICE=___
 FEED_BACK_HOST=___
-FEED_BACK_PORT=___
\ No newline at end of file
+FEED_BACK_PORT=___
+NEXT_PUBLIC_APP_URL=https://tarteel.quran.us.kg
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)